notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
Ever is an extra word in there. Can we remove it? <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=3>ReviewNB</a>_

Reply:
Yes. That's now taken care of.

Here's some longer blurb with a line break because I like long comments.

Reply:
Thanks for fixing it. This conversation is now resolved.

Reply:
<p>a</p>

Reply:
<p>b</p>

Reply:
<p>ab</p>

Reply:
<p>awe</p>

Reply:
<p>aqe</p>

Reply:
<p>aws</p>

Reply:
<p>aqs</p>

Reply:
<p>a</p>

Reply:
<p>a</p>

Reply:
<p>b</p>

Reply:
<p>ab</p>

Reply:
<p>This is a line</p><p>This is a new line</p><p><br></p><p>This is a new line after empty line above</p><p><br></p>